crates/re_types/definitions/rerun/blueprint/archetypes/viewport_blueprint.fbs

```diff
@@ -36,5 +36,17 @@ table ViewportBlueprint (
 
     // TODO(jleibs): This should come with an optional container id that specifies where to insert new space views.
     /// Whether or not space views should be created automatically.
+    ///
+    /// True if not specified, meaning that if the Viewer deems it necessary to add new Space Views to cover
```

Comment:
> but then you'd want "Default false if someone specifies a blueprint but default true otherwise"

Correct, but I wouldn't really describe that as "default true otherwise."  There are two separate concepts here:

For the purposes of querying the archetype, if there is a blueprint and it has no `auto_space_views` component, then the value should be interpreted as false.

However, if the user has not provided a blueprint and the viewer is going to create a blueprint for you, in the blueprint the viewer creates, it will **set** the value of `auto_space_views` to true. But it doesn't have to rely on a default-valued component to do so since it can be explicit about it's intent.

Reply:
I'd _really_ would like that case to just rely on a default True, but I get the practical problems with that, so I'll look into patching this up the way you describe it right away.